test/e2e/upgrades/apps/replicasets.go

```diff
@@ -87,7 +87,13 @@ func (r *ReplicaSetUpgradeTest) Test(f *framework.Framework, done <-chan struct{
 	}
 
 	ginkgo.By(fmt.Sprintf("Waiting for replicaset %s to have all of its replicas ready after upgrade", rsName))
-	framework.ExpectNoError(e2ereplicaset.WaitForReadyReplicaSet(c, ns, rsName))
+
+	err = e2ereplicaset.WaitForReadyReplicaSet(c, ns, rsName)
```

Comment:
@soltysh please upstream

Reply:
No need to upstream, this looks like debug commit - drop it. We can re-use `TestContext.DumpLogsOnFailure` if we need to.

Reply:
Dropped.